apidoc/Titanium/Media/AudioPlayer.yml

```diff
@@ -145,14 +146,36 @@ methods:
       - name: paused
         summary: Pass `true` to pause the current playback temporarily, `false` to unpause it.
         type: Boolean
+    deprecated:
+        since: "7.4.0"
+        notes: Use the cross-platform API [pause](Titanium.Media.AudioPlayer.pause) instead.
+    platforms: [iphone, ipad]
+
+  - name: seekToTime
+    parameters:
+      - name: time
+        summary: The time in milliseconds to seek to.
+        type: Number
+    summary: |
+        Moves the playback cursor and invokes the specified block when the seek 
+        operation has either been completed or been interrupted.
+    description: |
+        Use this method to seek to a specified time for the current player item and 
+        to be notified by the `seek` event when the seek operation is complete.  
     platforms: [iphone, ipad]
+    since: "7.4.0"
 
   - name: release
     summary: Stops buffering audio data and releases audio resources.
     description: |
         On Android, this method should be called when you are done streaming a given
         audio object, to release underlying resources, including buffered data.
-    platforms: [android]
+
+        On iOS, this method can be called for parity to release the current player
+        instance and it's observers. Note: If you are listening to the `progress` event,
+        you should remove and re-add the event listener.
```

Comment:
Question:
On iOS, what happens if you call an `AudioPlayer` object's methods/property after it has been released?

Reply:
The player will be re-initialized by the `[self player]` convenience method. But if Android behaves differently, I could block an instance from being re-initialized again!

Reply:
Currently, Android does the same via its `AudioPlayerProxy.getSound()`.

Reply:
Then we are fine for parity at that one? :-)

Reply:
Yes. :)